BET/Birkhoff.lean

```diff
@@ -34,10 +34,12 @@ variable {f g : α → ℝ} (hf : Integrable f μ) (hg : Integrable g μ)
 variable (x : α)
 variable (n : ℕ)
 
-#check birkhoffAverage ℝ T f n x
-
+/- define `A := { x | sup_n ∑_{i=0}^{n} f(T^i x) = ∞}`.
+def A := { x : α | ∃ n, ∀ C : ℝ, ∑ i in Finset.range n, f (T^[i] x) > C }
 
-/- define `A := { x | sup_n ∑_{i=0}^{n} f(T^i x) = ∞}`. -/
+define `A := { x | lim_n ∑_{i=0}^{n} f(T^i x) = ∞}`.
+-/
+def A := { x : α | Filter.Tendsto (fun n ↦ ∑ i in Finset.range n, f (T^[i] x)) Filter.atTop Filter.atTop }
```

Comment:
Is this correct mathematically? Doesn't this require some degree of convergence towards infinity? It seems to me that,
```
def A := { x : α | ∀ C : ℝ, ∃ n, ∑ i in Finset.range n, f (T^[i] x) > C }
```
includes some extra points since f is not bounded and can be negative and positive.

Reply:
Yes, it is not f that we ultimately want to use, I will need to update the definition to after we define the modified observable (the one that gives a monotonically increasing sequence)